.github/workflows/pr-close-signal.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
These three lines are in all of the YAML files and I'm not sure how GitHub will react

Reply:
GitHub will react just fine, since it conforms to the [YAML standard](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_start).